node/tests/TestUtilities.ts

```diff
@@ -1205,7 +1310,10 @@ export async function transactionTest(
     responseData.push(["zremRangeByLex(key8, -Inf, +Inf)", 0]); // key8 is already empty
 
     if (gte(version, "7.0.0")) {
-        baseTransaction.zadd(key14, { one: 1.0, two: 2.0 });
+        baseTransaction.zadd(key14, [
```

Comment:
old API shold work as well, so you can keep both

Reply:
I suppose, but why?

Reply:
For coverage